en/docs/administer/role-based-access-control.md

```diff
@@ -3,6 +3,32 @@
 Super admin can restrict each section in the admin portal based on the scopes. Please follow the
 below scopes  chart to define scopes.
 
+## Step 1 - Create a new role
+
+1. Sign in to WSO2 Management Console ( `https://<Server Host>:9443+<port offset>/carbon` )
+2. Click `Add` under `Users and Role`  and click on `Add New Role`.
+3. Give a name for the role and click `next`.
+4. In the permissions screen give the necessary permissions. For example we can enable the `Login` permission.
+
+## Step 2 - Create a new user
+
+1. Click `Add` under `Users and Role` and click on `Add New User`.
+2. Give an username, password and click `next`.
+3. In the Users of role screen select and assign the previosly created role for the new user.
+
+## Step 3 - Assign scopes for the user
+
+1. Sign in to WSO2 Management Console ( `https://<Server Host>:9443+<port offset>/admin` ) as the super admin or tenant admin
+2. Click `Scope Assignments` in the left sidebar and click on `Add scope mappings` .
+3. In the Provide role name text input give the role name which was previously created in step 1 and then click `next`.
+4. In the `Select Permissions` menu, select the `Custom scope assignments` option. And select the scopes that you want to assign for the newly created role. You can refer the following table when assign the scopes. For example If the admin wants to the newly created user to access the key managers settings in the admin portal he can assign `apim:keymanagers_manage`, `apim:tenantInfo`, and `apim:admin_settings`.
```

Comment:
```suggestion
4. In the `Select Permissions` menu, select the `Custom scope assignments` option. And select the scopes that you want to assign for the newly created role. You can refer the following table when assigning the scopes. For example, if the admin wants the newly created user to access the key managers settings in the admin portal he can assign `apim:keymanagers_manage`, `apim:tenantInfo`, and `apim:admin_settings`.
```

Reply:
Fixed in https://github.com/wso2/docs-apim/pull/7931/commits/655f63950910b3ccd0c81c822c3d0603d35fc393